packages/core/admin/admin/src/pages/HomePage/tests/index.test.js

```diff
@@ -57,1338 +59,53 @@ describe('Homepage', () => {
-        fill: #666687;
-      }
-
-      .c45:hover {
-        background-color: #f6f6f9;
-      }
-
-      .c45:active {
-        background-color: #eaeaef;
-      }
-
-      .c45 .c13 {
-        color: #32324d;
-      }
-
-      .c45 svg > g,
-      .c45 svg path {
-        fill: #32324d;
-      }
-
-      .c1 {
-        display: grid;
-        grid-template-columns: 1fr;
-      }
-
-      .c3 {
-        overflow-x: hidden;
-      }
-
-      .c7 {
-        display: grid;
-        grid-template-columns: repeat(12,1fr);
-      }
-
-      .c21 {
-        display: grid;
-        grid-template-columns: repeat(12,1fr);
-        gap: 24px;
-      }
-
-      .c8 {
-        grid-column: span 8;
-        max-width: 100%;
-      }
-
-      .c34 {
-        grid-column: span 4;
-        max-width: 100%;
-      }
-
-      .c44 {
-        grid-column: span 6;
-        max-width: 100%;
-      }
-
-      .c4:focus-visible {
-        outline: none;
-      }
-
-      .c48 path {
-        fill: #7289da !important;
-      }
-
-      .c49 > path:first-child {
-        fill: #ff4500;
-      }
-
-      .c52 > path:first-child {
-        fill: #4945ff;
-      }
-
-      .c52 > path:nth-child(2) {
-        fill: #fff;
-      }
-
-      .c52 > path:nth-child(4) {
-        fill: #9593ff;
-      }
-
-      .c50 path {
-        fill: #1da1f2 !important;
-      }
-
-      .c51 > path:first-child {
-        fill: #231f20;
-      }
-
-      .c51 > path:nth-child(2) {
-        fill: #fff9ae;
-      }
-
-      .c51 > path:nth-child(3) {
-        fill: #00aeef;
-      }
-
-      .c51 > path:nth-child(4) {
-        fill: #00a94f;
-      }
-
-      .c51 > path:nth-child(5) {
-        fill: #f15d22;
-      }
-
-      .c51 > path:nth-child(6) {
-        fill: #e31b23;
-      }
-
-      .c46 {
-        display: -webkit-box;
-        display: -webkit-flex;
-        display: -ms-flexbox;
-        display: flex;
-        -webkit-align-items: center;
-        -webkit-box-align: center;
-        -ms-flex-align: center;
-        align-items: center;
-        border: none;
-      }
-
-      .c46 svg {
-        width: 24px;
-        height: 24px;
-      }
-
-      .c46 span {
-        word-break: keep-all;
-      }
-
-      .c43 {
-        row-gap: 8px;
-        -webkit-column-gap: 16px;
-        column-gap: 16px;
-      }
-
-      .c16 {
-        word-break: break-word;
-      }
-
-      .c12 {
-        -webkit-align-items: flex-start;
-        -webkit-box-align: flex-start;
-        -ms-flex-align: flex-start;
-        align-items: flex-start;
-      }
-
-      .c22 {
-        -webkit-text-decoration: none;
-        text-decoration: none;
-      }
-
-      .c5 {
-        position: absolute;
-        top: 0;
-        right: 0;
-      }
-
-      .c5 img {
-        width: 9.375rem;
-      }
-
-      @media (max-width:68.75rem) {
-        .c8 {
-          grid-column: span 12;
-        }
-      }
+    expect(firstChild).toMatchSnapshot();
```

Comment:
Do you have the feeling this snapshot adds value for us?

Reply:
we don't test a lot in this component (e.g. the presence of each links) so rn I'd say yes but I could replace the snapshot with more coverage on links?

Reply:
If you have some time that would certainly be helpful I think :)